scenes/maingame/player/axe7C5F.tmp

```diff
@@ -0,0 +1,205 @@
+[gd_scene load_steps=13 format=3 uid="uid://cg1cnkcs84nfo"]
```

Comment:
working on a little axe swing animation here. However I was struggling with getting the animation to flip properly, not sure how to do it so I kept this animation but changed the default animation back